WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/PencilEditButton.swift

```diff
@@ -3,11 +3,28 @@ import SwiftUI
 /// Renders a button with the pencil system name image
 ///
 struct PencilEditButton: View {
+    @ScaledMetric private var scale: CGFloat = 1.0
+    private let size: CGFloat
```

Comment:
```suggestion
    @ScaledMetric private var size: CGFloat = Layout.editIconImageSize
```

Since we don't actually change the size anywhere, consider using the ScaledMetric directly and not having the parameter or multiplication. This works the same, but you can't pass the initial value for a scaled metric via the init, it needs to be known at compile time.

Reply:
Good point, updated on https://github.com/woocommerce/woocommerce-ios/pull/15685/commits/ebc08540213843cdd5a9025fb20cc87dc5535e94